app/views/physical_server/_main.html.haml

```diff
@@ -1,8 +1,4 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"), :items => textual_group_relationships}
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Power Management"), :items => textual_group_power_management}
-  .col-md-12.col-lg-6
-    TODO: other groups
+    TODO: add groups
```

Comment:
After https://github.com/ManageIQ/manageiq-ui-classic/pull/1283, `app/views/physical_server/_main.html.haml ` does not exist anymore.
Please rebase and then apply your changes.